# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.15; lt-2.4.6)
libtoolize -c -f; aclocal; autoheader; autoconf; automake -a -c
./configure --disable-cpp --disable-static --enable-unicode-properties --prefix=

# build
make

# install
install -d $DESTDIR/include
install -m 644 pcre.h $DESTDIR/include
install -d $DESTDIR/lib
install .libs/libpcre.so.1 $DESTDIR/lib
ln -sf libpcre.so.1 $DESTDIR/lib/libpcre.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 libpcre.pc $DESTDIR/lib/pkgconfig

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/pcre-build.pc << EOF
Name: pcre
Version: 8.41
Description: Perl Compatible Regular Expression library
Requires: $REQUIRES

devel=\\
/include/pcre.h \\
/lib/libpcre.so \\
/lib/pkgconfig/libpcre.pc

exec=\\
/lib/libpcre.so.1
EOF

exit

# patch
--- pcre-8.41.orig/configure.ac
+++ pcre-8.41/configure.ac
@@ -30,6 +30,8 @@
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AC_CONFIG_HEADERS(config.h)
 
+AM_MAINTAINER_MODE
+
 # This is a new thing required to stop a warning from automake 1.12
 m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 
# source
http://downloads.sourceforge.net/pcre/pcre-8.41.tar.gz
